### PR TITLE
Buffs Banhammer

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl:
rscadd: Buffs banhammer suicide
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
:+1: :clap: :clapper: :beers:

**Comment by PKPenguin321** [comment]
Brutal

**Comment by Mercenaryblue** [comment]
:clap: :clap: :clap:

**Comment by Man-Shroom-** [comment]
how long is 1440 or whatever

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -13,8 +13,10 @@
 	attack_verb = list("banned")
 
 /obj/item/weapon/banhammer/suicide_act(mob/user)
-		user.visible_message("<span class='suicide'>[user] is hitting \himself with the [src.name]! It looks like \he's trying to ban \himself from life.</span>")
-		return (BRUTELOSS|FIRELOSS|TOXLOSS|OXYLOSS)
+	user.visible_message("<span class='suicide'>[user] is hitting \himself with the [src.name]! It looks like \he's trying to ban \himself from life.</span>")
+	if(user.client)
+		AddBan(user.ckey, user.computer_id, "Killed themselves with the banhammer", "banhammer", 1, 1440)
```
Comment (Supermichael777) [inline code/game/objects/items/weapons/weaponry.dm]: make it 1 min

**Comment by KazeEspada** [comment]
Only issue is notes.

**Comment by Dorsisdwarf** [comment]
Ban raisin is pretty clear

**Comment by Mercenaryblue** [comment]
Add a server-wide message when somebody does this. And/or add changelog.

**Comment by Shapsy** [comment]
I can't wait to trick people with this

**Comment by Jordie0608** [comment]
I don't really like this, ban tables shouldn't get messed with.
You could just fake the ban messages and drop the client.

**Comment by TechnoAlchemisto** [comment]
holy shit lmao

**Comment by KazeEspada** [comment]
Im all for applying a ban to someone who suicides with this, but actually adding it to the notes fucks with the notes.

**Comment by Fox-McCloud** [comment]
>I don't really like this, ban tables shouldn't get messed with.
You could just fake the ban messages and drop the client.

This is what I kinda feel. Just make it emulate the message as closely as possible and close their client.

**Comment by KorPhaeron** [comment]
I've amassed too many :tada: you serious database minded admins can't stop me now

**Comment by Jordie0608** [comment]
Don't make us get the Niknak out

**Comment by Niknakflak** [comment]
yourang

**Comment by KorPhaeron** [comment]
@Cheridan 

Up to you

**Comment by PKPenguin321** [comment]
*prod*

**Comment by Iamgoofball** [comment]
please make it 1 minute

**Pull request closed without merging**